style.css

```diff
@@ -0,0 +1,379 @@
+html {
+  align-items: center;
+  width: 100%;
+  display: flex;
+}
+
+body {
+  background: #e9e8e8b6;
+  position: absolute;
+  width: 100%;
+  border: 1px;
+  margin: 20px;
+  left: 0;
+  top: 0;
+  align-items: center;
+}
+
+header {
+  width: 375px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: basline;
+  margin: 20px;
+  padding-left: 0;
+}
+
+header p {
+  color: #6070ff;
+  width: 140px;
+  height: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 29px;
+  line-height: 20px;
+  display: flex;
+  text-align: start;
+}
+
+header i {
+  font-size: 2.2rem;
+  margin-right: 50px;
+}
+
+button a {
+  text-decoration: none;
+}
+
+.headline a {
+  text-decoration: none;
+}
+
+.wraper {
+  background-color: white;
+  width: 380px;
+  border-radius: 0 0 0 80px;
+}
+
+.headline {
+  background-image: url(/header-shapes\ mobile.png);
+  background-size: 375px;
+  background-repeat: no-repeat;
+  background-position: left + 10px;
+  background-position-y: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex start;
+  padding: 114px 24px;
+  gap: 12px;
+  width: 350px;
+  height: 424px;
+  left: calc(45% - 375px / 2);
+  top: 20px;
+  border-radius: 0 0 8px 0;
+}
+
+h1 {
+  width: 300px;
+  height: 150px;
+  margin: 1px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.details {
+  width: 300px;
+  height: 140px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 28px;
+  color: #344563;
+  margin-bottom: 40px;
+  margin-top: -1px;
+}
+
+h4 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  text-decoration: none;
+  font-style: normal;
+  color: #7f8cff;
+  margin: 0;
+}
+
+.social ul {
+  display: flex;
+  width: 150px;
+  height: 40px;
+  justify-content: space-between;
+  font-size: 30px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ ] Additionally, we have to try as much as possible to stick to [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). This includes the padding, margin, font, color, etc. Therefore, I suggest that you ensure the icons have the same color that matches that of the Figma design 🥰 .

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/220320072-fcd3b420-cb81-41cc-b007-9c868b534dd9.png)